lib/core/src/server/build-dev.js

```diff
@@ -153,7 +154,9 @@ Would you like to run Storybook on port ${port} instead?`,
     const [stats] = await Promise.all([webpackValid, serverListening]);
     const proto = program.https ? 'https' : 'http';
     const address = `${proto}://${program.host || 'localhost'}:${port}/`;
-    logger.info(`Storybook started on => ${chalk.cyan(address)}\n`);
+    const networkAddress = `${proto}://${ip.address()}:${port}/`;
+    logger.info(`Storybook started on => ${chalk.cyan(address)}`);
```

Comment:
Why was the `\n` removed from this line?

Reply:
the `\n` prints an empty `info` line, removing it keeps these 2 lines adjacent in the console. note that the new network address line prints a `\n` so there's still an empty `info` line in the console after the 2 address lines

Reply:
updated `yarn.lock` with yarn 1.9.4